frontend/packages/dev-console/src/components/monitoring/queries.ts

```diff
@@ -85,7 +94,7 @@ export const monitoringDashboardQueries: MonitoringQuery[] = [
       `topk(25, sort_desc(sum(rate(container_network_receive_bytes_total{ container="POD", pod!= "", namespace = '<%= namespace %>'}[5m]) + rate(container_network_transmit_bytes_total{ container="POD", pod!= "", namespace = '<%= namespace %>'}[5m])) BY (namespace, pod)))`,
     ),
     chartType: GraphTypes.line,
-    title: 'Network Received ',
```

Comment:
why change this?

Reply:
Query includes both Received & Transmitted network so changed the title as per query.

Reply:
Are you sure about that? If so then that seems like a bug because there is the separate metric `container_network_transmit_bytes_total`.

Reply:
Separated received & transmitted bandwidth.